### PR TITLE
Overview: review and make minor changes

### LINKED ISSUE #1228: [Docs Review] Overview
Parent Issue vitessio/website#1222 
Vitess github url: https://github.com/vitessio/website

### Pages to review:

- [x] [Overview](https://vitess.io/docs/16.0/overview/) - Need to add "Why Vitess", but we have a separate issue for it #1217
- [x] [What Is Vitess](https://vitess.io/docs/15.0/overview/whatisvitess/)
- [x] [Architecture](https://vitess.io/docs/15.0/overview/architecture/)
- [x] [Supported Databases](https://vitess.io/docs/15.0/overview/supported-databases/) - need to update the version here to 8.0 as part of https://github.com/vitessio/vitess/pull/11989 
- [x] [Scalability Philosophy](https://vitess.io/docs/15.0/overview/scalability-philosophy/)
- [x] [Cloud Native](https://vitess.io/docs/15.0/overview/cloud-native/)
- [x] [History](https://vitess.io/docs/15.0/overview/history/)

### What to do 

- [x] Make sure the content reflect the current state and facts are up-to-date.
- [x] Add details if needed either in form of new section on current page or adding new page.
- [x] Add links/Urls on page are working
- [x] Deprecate flags should be marked as 'deprecated'
- [x] Review is done for both V15 and V16

Please note that we are not going to focus of restructuring the website content as part of this project. It is something we will do as a separate project later on.

### PR DESCRIPTION
**Pull request by deepthi**
Mostly cosmetic/minor changes.
The biggest change is to replace language about orchestrator integration with "native failure detection and recovery".
I've also added a link to vitess-operator on the cloud-native page.

Fixes #1228

### CODE REVIEW

File: content/en/docs/16.0/overview/cloud-native.md [hunk 1]
```diff
@@ -3,13 +3,15 @@ title: Cloud Native
...
+Vitess is well-suited for Cloud deployments because it enables databases to incrementally add capacity. The easiest way to run Vitess in the Cloud is via Kubernetes.
 
 ## Vitess on Kubernetes
 
-Kubernetes is an open-source orchestration system for Docker containers, and Vitess can run as a Kubernetes-aware cloud native distributed database.
+Kubernetes is an open-source orchestration system for containerized applications, and Vitess can run as a Kubernetes-aware cloud native distributed database.
 
-Kubernetes handles scheduling onto nodes in a compute cluster, actively manages workloads on those nodes, and groups containers comprising an application for easy management and discovery. This provides an analogous open-source environment to the way Vitess runs in YouTube, on the predecessor to Kubernetes.
+Kubernetes handles scheduling onto nodes in a compute cluster, actively manages workloads on those nodes, and groups containers comprising an application for easy management and discovery. This provides an analogous open-source environment to the way Vitess ran in YouTube on Borg, the predecessor to Kubernetes.
```
Comment (rsajwani) [inline content/en/docs/16.0/overview/cloud-native.md:12]: minor:
[borg] (https://kubernetes.io/blog/2015/04/borg-predecessor-to-kubernetes/)

File: content/en/docs/16.0/overview/scalability-philosophy.md [hunk 2]
```diff
@@ -55,9 +55,9 @@ As for atomicity, the following levels are supported:
...
-* High availability: Vitess integrates with Orchestrator, which is capable of performing a failover to a new primary within seconds of failure detection. This is usually sufficient for most applications.
-* Low-latency geographically distributed writes: This is one case that is not addressed by Vitess. The current recommendation is to absorb the latency cost of long-distance round-trips for writes. If the data distribution allows, you still have the option of sharding based on geographic affinity. You can then setup primarys for different shards to be in different geographic location. This way, most of the primary writes can still be local.
+* Scalability: There are situations where active-active gives you a little bit of additional runway. However, since all writes have to eventually be applied to all writable servers, it’s not a sustainable strategy. Vitess addresses this problem through sharding, which can scale indefinitely.
+* High availability: Vitess has native capability for detecting primary failures and performing a failover to a new primary within seconds of failure detection. This is usually sufficient for most applications.
```
Comment (rsajwani) [inline content/en/docs/16.0/overview/scalability-philosophy.md:59]: should we say VTorc in barckets here.

**Review by rsajwani — APPROVED** [review]

**Review by GuptaManan100 — APPROVED** [review]
Other than Rameez's comments, I don't have anything extra.

File: content/en/docs/16.0/overview/cloud-native.md (hunk 1 above)
Comment (deepthi) [reply content/en/docs/16.0/overview/cloud-native.md:12]: I debated adding a link and decided it is not worth it. It's easy enough to google.

File: content/en/docs/16.0/overview/scalability-philosophy.md (hunk 2 above)
Comment (deepthi) [reply content/en/docs/16.0/overview/scalability-philosophy.md:59]: I wanted to, but it was hard to fit it into this sentence in a good (readable) way. And anyway this is supposed to be high-level so I felt that it is ok to not name the specific component.

**Pull request merged**